docs/src/Guides/01 Getting Started.md

```diff
@@ -56,12 +56,13 @@ Now let's get the library installed.
 Now let's get a basic bot going, for your code, you'll want something like this:
 
 ```python
-from interactions import Client, Intents, listen
+from interactions import Client, listen
 
-bot = Client(intents=Intents.DEFAULT)
-# intents are what events we want to receive from discord, `DEFAULT` is usually fine
+bot = Client(
+    token="Put your token here"
+)
```

Comment:
i like when intents was apart of the client ngl

Reply:
``client = Client(token=IS_PROD, intents=Intents.ALL)``

This is the proper way you do it.

Reply:
For the intents: it may offer nothing *at that moment*, but having it there both explains that intents are important in interactions.py (as they *should* be) and should be set by the user whenever possible. At the very least, I see no harm to their addition - perhaps a link to the events guide would be a good idea in case someone was curious, though.

For the token... it's partially because that's how other libraries do it, and partially because they allow for making the bot in scenarios where you may not have the token yet, IE if you're making it outside of the main file and then starting the bot in the main one. The example in the repository was... well, frankly, I'm not sure why it's there, because it should be removed 😅. That was mostly for testing v5 when we were making it.

Honestly, I think my biggest complaint is that the init was changed for little to no gain - it isn't exactly unclear in its intent as is. Though keeping the token there is... alright, I suppose.